src/sdk.ts

```diff
@@ -68,21 +67,40 @@ export class ValoremSDK {
     }
 
     this.clearinghouse = new ClearinghouseContract({
-      publicClient: this.publicClient as PublicClient<Transport, Chain>,
-      walletClient: this.walletClient as WalletClient<
-        Transport,
-        Chain,
-        Account
-      >,
+      publicClient: this.publicClient,
+      walletClient: this.walletClient,
     });
 
     this.seaport = new SeaportContract({
-      publicClient: this.publicClient as PublicClient<Transport, Chain>,
-      walletClient: this.walletClient as WalletClient<
-        Transport,
-        Chain,
-        Account
-      >,
+      publicClient: this.publicClient,
+      walletClient: this.walletClient,
     });
   }
+
+  public getWebTaker() {
+    if (this.webTaker === undefined)
+      throw new Error(
+        'Failed to get WebTaker. Please initialize Valorem SDK with a wallet client to access WebTaker.',
+      );
+
+    return this.webTaker;
+  }
+
+  public getTaker() {
+    if (this.taker === undefined)
+      throw new Error(
+        'Failed to get Taker. Please initialize Valorem SDK with a wallet client to access Taker.',
+      );
+
+    return this.taker;
+  }
+
+  public getMaker() {
+    if (this.maker === undefined)
+      throw new Error(
+        'Failed to get Maker. Please initialize Valorem SDK with a wallet client to access Maker.',
+      );
+
+    return this.maker;
+  }
```

Comment:
Could use dynamic `get()` dynamic object property, to keep the usage for consumer simpler.

```typescript
get taker() { ... }
get maker() { ... }
get webTaker() { ... }
```